saos-import/src/main/java/pl/edu/icm/saos/importer/notapi/common/JudgmentObjectDeleter.java

```diff
@@ -67,4 +78,46 @@ public void deleteJudgmentsWithoutRawSourceJudgment(Class<? extends Judgment> ju
         }
     }
     
+    @Transactional
+    public void deleteMeansOfAppealWithoutJudgments(CourtType courtType) {
```

Comment:
In my view the better idea would be to put these 2 new methods into a separate service as it has been done in the case of deleting sc objects. See: ScObjectDeleter.